packages/typespec-vscode/src/const.ts

```diff
@@ -0,0 +1,18 @@
+import vscode from "vscode";
+
+export const StartFileName = "main.tsp";
+
+export const PreDefinedEmitterPickItems: Record<string, vscode.QuickPickItem> = {
```

Comment:
this is not a global const. Please move it to emitter file or folder. And why not getting it from config so that we don't need to hard code it here?

Reply:
removed to emitter file.